src/main/java/uk/gov/hmcts/reform/hmc/api/controllers/HearingsController.java

```diff
@@ -37,6 +39,8 @@ public class HearingsController {
     public static final String PROCESSING_REQUEST_AFTER_AUTHORIZATION =
             "processing request after authorization";
 
+    public static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
```

Comment:
Move to constant file above two constants

Reply:
Moved